politeiad/plugins/comments/comments.go

```diff
@@ -55,6 +59,12 @@ const (
 	// SettingAllowExtraData is the default value of the bool flag which
 	// determines whether posting extra data along with the comment is allowed.
 	SettingAllowExtraData = false
+
+	// SettingVotesPageSize is the default maximum number of comment votes
+	// that can be returned at any one time. It defaults to 200 to limit the
+	// comment votes route payload size to be ~80KB, as each comment vote size
+	// is expected to be around 400 bytes which means 200 * 0.4 = 80KB.
```

Comment:
Nice documentation.  I think we can increase this to 1 MiB.

The way I chose this number is pretty arbitrary.  Reading 1 MiB of data into memory is fine and returning 1 MiB of data is not that expensive from a network latency perspective.